packages/react-input/src/components/Input/Input.stories.tsx

```diff
@@ -1,32 +1,98 @@
 import * as React from 'react';
-import { makeStyles } from '@fluentui/react-make-styles';
+import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
 import { Input } from './Input';
-import { useId } from '@fluentui/react-utilities';
+import { getNativeElementProps, useId } from '@fluentui/react-utilities';
+import { InputProps } from './Input.types';
+import { ArgTypes } from '@storybook/react';
+// prevent terrible reload times by using deep imports :(
+import Search16Regular from '@fluentui/react-icons/lib/esm/components/Search16Regular';
+import Search20Regular from '@fluentui/react-icons/lib/esm/components/Search20Regular';
+import Search24Regular from '@fluentui/react-icons/lib/esm/components/Search24Regular';
+import Dismiss16Regular from '@fluentui/react-icons/lib/esm/components/Dismiss16Regular';
+import Dismiss20Regular from '@fluentui/react-icons/lib/esm/components/Dismiss20Regular';
+import Dismiss24Regular from '@fluentui/react-icons/lib/esm/components/Dismiss24Regular';
 
 const useStyles = makeStyles({
   container: {
     display: 'flex',
     flexDirection: 'column',
     gap: '20px',
-    width: '300px',
+    padding: '20px',
   },
+  storyFilledBackground: theme => ({ background: theme.alias.color.neutral.neutralBackground3 }),
 });
 
-export const InputExamples = () => {
+const icons = {
+  search: { small: Search16Regular, medium: Search20Regular, large: Search24Regular },
+  dismiss: { small: Dismiss16Regular, medium: Dismiss20Regular, large: Dismiss24Regular },
+};
+
+export const InputExamples = (
+  args: Partial<InputProps> & React.InputHTMLAttributes<HTMLInputElement> & { storyFilledBackground: boolean },
+) => {
   const styles = useStyles();
   const inputId1 = useId();
+  // pass native input props to the internal input element and custom props to the root
+  const { storyFilledBackground, ...rest } = args;
+  const inputProps = getNativeElementProps('input', rest, ['size']);
+  const props: Partial<InputProps> = { input: inputProps };
+  for (const prop of Object.keys(rest) as (keyof InputProps)[]) {
+    if (!(inputProps as Partial<InputProps>)[prop]) {
+      props[prop] = rest[prop];
+    }
+  }
+  const SearchIcon = icons.search[props.size!];
+  const DismissIcon = icons.dismiss[props.size!];
+
   return (
-    <div className={styles.container}>
-      <Input />
+    <div className={mergeClasses(styles.container, storyFilledBackground && styles.storyFilledBackground)}>
+      <Input {...props} />
       <div>
         <label htmlFor={inputId1}>with a label</label>
-        <Input id={inputId1} />
+        <Input {...props} id={inputId1} />
       </div>
+      <Input {...props} insideStart={<SearchIcon />} insideEnd={<DismissIcon />} />
```

Comment:
Not necessarily something to change now and in this PR, but in the future I think all our storybook examples should be accessible by default (i.e. have a visible label and no placeholder), with potentially one example of less-accessible variations like not having a visible label.

Reply:
Yup I plan to follow up on this when I make a more finished version of the stories